### ISSUE TITLE
LineEdit.editable and TextEdit.readonly should have identical naming scheme.

### ISSUE THREAD

**Issue by sviatoslav-vilkovych** [godotengine/godot-proposals#1792]
**Describe the project you are working on:**
It is a personal project for the university. I am making a financial calculator. Not a perfect case for Godot, but I want to learn Godot so I am trying to use it everywhere.
**Describe the problem or limitation you are having in your project:**
I replaced TextEdit with LineEdit and noticed that present in TextEdit [**readonly**](https://docs.godotengine.org/en/stable/classes/class_textedit.html#class-textedit-property-readonly) property is not present in LineEdit. Instead of it, LineEdit has [**editable**](https://docs.godotengine.org/en/stable/classes/class_lineedit.html#class-lineedit-property-editable). Looking into the documentation of both, I think they identical:

> ...existing text cannot be modified and new text cannot be added...

**Describe the feature / enhancement and how it helps to overcome the problem or limitation:**
I propose to have a single naming for this functionality unless such a different naming scheme intended for something else (but that's up to Godot team, I just want to bring awareness).
**Describe how your proposal will work, with code, pseudocode, mockups, and/or diagrams:**
My proposed enhancement will help clients of Godot to learn faster and better understand API.
When writing your code, you will know that either `editable` or `readonly` responsible for making node **uneditable**, so it will be possible to have an array of nodes and disable input to all of them when needed.
```
var array_of_lineEditsAndTextEdits = [textEdit1, textEdit2, lineEdit1, lineEdit2]
for node in array_of_lineEditsAndTextEdits:
   node.readonly = true # or it can be node.editable = false
```
**If this enhancement will not be used often, can it be worked around with a few lines of script?:**
Of course, this can be avoided by having an if-statement in this loop or splitting the array into two.
**Is there a reason why this should be core and not an add-on in the asset library?:**
This enhancement proposal related to naming in Godot and can not be changed by the asset library.

Thank you for your attention!

**Comment by EricEzaM** [comment]
I have discussed this briefly with @Paulb23 and I believe it will be addressed in a TextEdit/LineEdit cleanup pass.

https://github.com/godotengine/godot/pull/42600#discussion_r502795505

**Comment by Paulb23** [comment]
Fixed by godotengine/godot/pull/50371.
